src/modals/confirmation-modal/operations-confirmation/hooks/use-estimations.hook.ts

```diff
@@ -13,10 +13,10 @@ export const useEstimations = (sender: WalletAccountInterface, opParams: ParamsW
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
-    const subscription = from(
-      tezos$.getValue().estimate.batch(opParams.map(param => ({ ...param, source: sender.publicKeyHash })))
-    )
+    const subscription = of(null)
```

Comment:
don't like this.  
rewrite with
```
tezos$.getValue().estimate.batch(...)
```

Reply:
we replaced old `tezos$` method with observable so i decided to use just `tezos$`. in current code `tezos$` doesn't have method `getValue()`